locations/spiders/mbplc_gb.py

```diff
@@ -0,0 +1,60 @@
+from locations.categories import Categories
+from locations.storefinders.woosmap import WoosmapSpider
+
+
+class MbplcGBSpider(WoosmapSpider):
+    name = "mbplc_gb"
+    key = "woos-19f74f73-0b20-3571-b9ef-706ad3dbad27"
+    origin = "https://www.mbplc.com/"
+
+    brand_mapping = {
+        "Sizzling Pubs": {"brand": "Sizzling Pubs", "brand_wikidata": "Q118314187"},
+        "All Bar One": {"brand": "All Bar One", "brand_wikidata": "Q4728624"},
+        "Browns Restaurants": {"brand": "Browns Restaurants", "brand_wikidata": "Q4976672"},
+        "Ember Inns": {"brand": "Ember Inns", "brand_wikidata": "Q116272278"},
+        "Harvester": {"brand": "Harvester", "brand_wikidata": "Q5676915"},
+        "Miller & Carter": {"brand": "Miller & Carter", "brand_wikidata": "Q87067401"},
+        "Nicholsons": {"brand": "Nicholsons", "brand_wikidata": "Q113130666"},
+        "O'Neill's": {"brand": "O'Neill's", "brand_wikidata": "Q7071905"},
+        "Premium Country Pubs": {
+            "brand": "Premium Country Pubs",
+            "brand_wikidata": "Q118606900",
+            "extras": Categories.PUB.value,
+        },
+        "Stonehouse": {"brand": "Stonehouse", "brand_wikidata": "Q78192049"},
+        "Toby Carvery": {"brand": "Toby Carvery", "brand_wikidata": "Q7811777"},
```

Comment:
https://github.com/search?q=repo%3Aalltheplaces%2Falltheplaces+Q7811777&type=code - Q7811777 - 2x existing spiders to review

Reply:
https://alltheplaces-data.openaddresses.io/map.html?show=https%3A%2F%2Falltheplaces-data.openaddresses.io%2Fruns%2F2024-08-17-13-32-02%2Foutput%2Ftoby_carvery_gb.geojson - 150 locations